### PR TITLE
fix : #1418 Mobile view error when text is to long

### LINKED ISSUE #1418: Mobile view error when text is to long
### Description

When viewing for example eddies profile the website looks like in the screenshot below. In my opinion it is because of the long text for the yt channel. 

### Screenshots

2F4AB263-8912-46AB-8E09-7CE7D1073820

### Additional information

Device: iPhone 13 mini
Browser: Safari latest version

### PR DESCRIPTION
**Pull request by thevinitgupta**
## Fixes Issue

Closes #1418 

## Changes proposed

[x] - Link Button with long text overflows from screen

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Initial : 
initial_eddie

After changes :
final_eddie 

## Note to reviewers
This is my first contribution to the project. I have tried my best to follow the contribution guidelines.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1426"></a>

### CODE REVIEW

**Review by donmarvex — COMMENTED** [review]
Looks good to me! Nice work @thevinitgupta

**Review by emmalearnscode — APPROVED** [review]
This works nicely 👍 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by thevinitgupta** [comment]
Thanks @emmalearnscode 
I wanted to ask when will the PR be merged?

**Comment by emmalearnscode** [comment]
> Thanks @emmalearnscode
> I wanted to ask when will the PR be merged?

Once it’s got two maintainer approvals. As all our maintainers are volunteers it could be a few days

**Comment by 87prashant** [comment]
Nice work @thevinitgupta 👏
Could you also optimize the `ShareProfile` component for mobile?
Screenshot 2022-07-03 195217

**Review by 87prashant — APPROVED** [review]
Nice work 👍

**Comment by emmalearnscode** [comment]
> Nice work @thevinitgupta 👏
> Could you also optimize the `ShareProfile` component for mobile?
> Screenshot 2022-07-03 195217

Please raise a separate issue for this as it’s not within the scope of this pull request

**Comment by thevinitgupta** [comment]
> > Thanks @emmalearnscode
> > I wanted to ask when will the PR be merged?
> 
> Once it’s got two maintainer approvals. As all our maintainers are volunteers it could be a few days

Okay great.

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍

**Comment by 87prashant** [comment]
> Please raise a separate issue for this as it’s not within the scope of this pull request

Ok

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Great, Thanks for your Contribution! 🚀

**Pull request merged**
